node-api/handlers/beacon/genesis.go

```diff
@@ -35,11 +37,23 @@ func (h *Handler) GetGenesis(_ handlers.Context) (any, error) {
 	if len(genesisRoot) == 0 {
 		return nil, types.ErrNotFound
 	}
+
+	genesisForkVersion, err := h.backend.GenesisForkVersion(utils.Genesis)
+	if err != nil {
+		return nil, err
+	}
+	genesisForkVersionHex := "0x" + hex.EncodeToString(genesisForkVersion[:])
```

Comment:
this isn't needed just return `genesisForkVersion.String()`. The type already handles this.

Reply:
yeah, done